R/as_wide_table.R

```diff
@@ -37,7 +37,7 @@ as_wide_table <- function(austraits){
 
     # The contexts table needs the contexts collapsed to one context name per site
   austraits %>% 
-    join_contexts(collapse_context = TRUE) -> austraits
+    join_contexts_old(collapse = TRUE) -> austraits
```

Comment:
Why is this still available? Should we remove it?

Reply:
The `as_wide_table` function requires the old version of `join_contexts`. It isn't too hard to weave the new `join_context_properties` in, but then the output is different and to maintain the austraits.build API I don't think we want to mess with that right now.